assist_microphone/build.yaml

```diff
@@ -0,0 +1,10 @@
+---
+build_from:
+  amd64: ghcr.io/home-assistant/amd64-base-debian:bookworm
+  aarch64: ghcr.io/home-assistant/aarch64-base-debian:bookworm
+  armv7: ghcr.io/home-assistant/armv7-base-debian:bookworm
```

Comment:
Any specific reason we built this around Debian and not our default Alpine?

Reply:
I use the `webrtc-noise-gain` package for audio enhancements, which has a pre-compiled wheel for glibc.